tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+skipsdist=True
+usedevelop=True
+
+envlist=dj{18,111}
+
+[testenv]
+install_command=pip install -e ".[testing]" -U {opts} {packages}
+commands=
+     coverage erase
+     coverage run manage.py test {posargs}
+
+deps=
+    coverage==4.5.1
```

Comment:
If you move these into an `extras_require` section as described above, you can remove this `deps` section, as the `pip install -e ".[testing]"` will install them.